cmd/snap/inhibit.go

```diff
@@ -25,12 +25,35 @@ import (
 	"os"
 	"time"
 
+	"github.com/godbus/dbus"
 	"github.com/snapcore/snapd/cmd/snaplock/runinhibit"
+	"github.com/snapcore/snapd/dbusutil"
 	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/logger"
 	"github.com/snapcore/snapd/progress"
 	"github.com/snapcore/snapd/usersession/client"
 )
 
+func tryNotifyRefreshToSnapDesktopIntegration(snapName string) (bool, error) {
```

Comment:
nitpick: this should go closer to the Flow functions

for consistency this should probably be called tryNotifyRefreshViaSnapDesktopIntegrationFlow

Reply:
Done.